docs/src/components/Tabs.jsx

```diff
@@ -0,0 +1,44 @@
+import React, { useState } from 'react';
+import PropType from 'prop-types';
+
+export const TabItem = (props) => {
+	return props.children;
+};
+TabItem.propTypes = {
+	children: PropType.oneOfType([PropType.string, PropType.array]),
+};
+
```

Comment:
So for accessibility we should be using aria roles here to describe the tab components.  The structure should look something like
```
<div class="tabs">
  <div class="tabs__tablist" role="tablist">
    <button class="tabs__button" role="tab" id="tab_1" aria-controls="tabpanel_1">tab label</button>
  </div>
  <div class="tabs__panel" role="tabpanel" id="tabpanel_1" aria-labelledby="tab_1">tab content</div>
</div>
```

Reply:
Naw, that stuff is good - and we should be documenting that somewhere for us as we start to build components for NCIDS (checklist for interactive things etc..)

Reply:
updated here for aria / role - check to make sure what I'm doing is cool. As mentioned just using the prop for styling to switch values.